armi/reactor/converters/uniformMesh.py

```diff
@@ -400,13 +405,13 @@ def makeAssemWithUniformMesh(
             if len(typeHeight) > 1:
                 if sourceBlock:
                     totalHeight = sum(typeHeight.values())
-                    runLog.extra(
+                    runLog.debug(
                         f"Multiple XS types exist between {bottom} and {topMeshPoint}. "
                         f"Using the XS type from the largest region, {xsType}"
                     )
                     for xs, h in typeHeight.items():
                         heightFrac = h / totalHeight
-                        runLog.extra(f"XSType {xs}: {heightFrac:.4f}")
+                        runLog.debug(f"XSType {xs}: {heightFrac:.4f}")
```

Comment:
I think we need more than this.

Just because a single log line was useful to someone 8 years ago, doesn't mean anyone has looked at it in the last 7 years.

And I _definitely_ think we have a lot of `debug` log lines masquerading as `extra` and even `info`.

Nice!

Reply:
Thanks! Unfortunately, I think I'm the one who initially put this in as an `extra` just a couple months ago. I didn't realize how much obnoxious output it creates until I started testing with `detailedAxialExpansion: true`